packages/sdk/src/oracle/client.ts

```diff
@@ -0,0 +1,32 @@
+import Store, { Emit } from "./store";
+import type { state } from "./types";
+import { Inputs } from "./types/state";
+import { Signer } from "./types/ethers";
+
+export class Client {
```

Comment:
this is the client exposed to the frontend, currently only 2 functions, one to set your active wallet, the other to set the active request, ie which oracle request we are on